chemprop/nn/metrics.py

```diff
@@ -103,7 +105,9 @@ def forward(
         lt_mask: Tensor,
         gt_mask: Tensor,
     ):
-        squared_errors = super()._calc_unreduced_loss(preds, targets, mask, lt_mask, gt_mask)
+        squared_errors = super()._calc_unreduced_loss(
+            preds.unsqueeze(-1), targets, mask, lt_mask, gt_mask
```

Comment:
same as here

Reply:
Please see the response at https://github.com/chemprop/chemprop/pull/959/files#r1699193823.